assets/src/domain/eventEditor/ui/ticketAssignmentsManager/useTicketAssignmentsManager/index.tsx

```diff
@@ -1,25 +1,29 @@
 import React, { useEffect, useCallback } from 'react';
 import { __ } from '@wordpress/i18n';
-import { TicketAssignmentsManager, ErrorMessage } from './components';
+import { TicketAssignmentsManager, ErrorMessage } from '../components';
 import { ButtonProps } from 'antd/lib/button';
 import { ModalFunc } from 'antd/lib/modal/confirm';
 import { SaveOutlined } from '@ant-design/icons';
-import { BaseProps, TicketAssignmentsManager as TAM } from './types';
+import { BaseProps, TicketAssignmentsManager as TAM } from '../types';
 import useModal from '@appLayout/modal/useModal';
-import { withContext } from './context';
-import { useDataState, useOnSubmitAssignments } from './data';
+import { withContext } from '../context';
+import { useDataState, useOnSubmitAssignments } from '../data';
+
+import './styles.scss';
 
 const useTicketAssignmentsManager = (): TAM => {
 	const modal = useModal();
 	const submitAssignments = useOnSubmitAssignments();
 	let openModal: ReturnType<ModalFunc>;
 
-	const assignTicketsToDate: TAM['assignTicketsToDate'] = ({ datetimeId }) => {
-		showModal({ assignmentType: 'forDate', entityId: datetimeId });
+	const assignTicketsToDate: TAM['assignTicketsToDate'] = ({ datetimeId, name }) => {
+		const title = __('Ticket Assignment Manager for Datetime: ') + `${datetimeId} - ${name}`;
```

Comment:
other languages may not want those variables on the right side of the string so we always need to use i18n functions like `__()`:

```suggestion
		const title = sprintf(
			__('Ticket Assignment Manager for Datetime: %s - %s'),
			datetimeId,
			name
		);
```